rand_core/CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## [0.1.0] - TODO - date
+## [0.1.1] - Unreleased
+- Enable the `std` feature by default. (#409)
+- Change `BlockRng64::inner` and add `BlockRng64::inner_mut` to mirror `BlockRng`. (#419)
+- Add `BlockRng{64}::index` and `BlockRng{64}::generate_and_set`. (#374, #419)
+- Add `AsMut<[Self::Item]>` bound to `BlockRngCore::Results` (future-proofing). (#419)
```

Comment:
Since this is technically a breaking change, I think it would be better to rephrase as "Change `BlockRngCore::Results` bound to also require `AsMut<[Self::Item]>`".

And since we have two small breaking changes shall we call this 0.2.0 instead?